mobile/src/components/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -19,27 +21,45 @@ const SignUpForm: FC<Props> = ({ onSubmit }) => {
   });
 
   return (
-    <>
-      <Text>Sign Up</Text>
+    <View style={styles.container}>
+      <Image source={require('../../../../assets/images/logo.png')} />
+      <View style={styles.circle_1} />
+      <View style={styles.circle_2} />
+      <Text style={styles.title}>Create an account</Text>
       <View>
-        <Input
-          label="Email"
-          placeholder="Enter your email"
-          name="email"
-          control={control}
-          errors={errors}
-        />
-        <Input
-          label="Password"
-          placeholder="Enter your password"
-          name="password"
-          control={control}
-          errors={errors}
-        />
-        <Button label="Sign up" onPress={handleSubmit(onSubmit)} />
+        <View style={styles.inputWrapper}>
+          <Input
+            label="Full Name"
+            placeholder="Enter your full name"
+            name="fullName"
+            control={control}
+            errors={errors}
+          />
+        </View>
+        <View style={styles.inputWrapper}>
+          <Input
+            label="Email"
+            placeholder="Enter your email"
+            name="email"
+            control={control}
+            errors={errors}
+          />
+        </View>
+        <View style={styles.inputWrapper}>
+          <Input
+            label="Password"
+            placeholder="Enter your password"
+            name="password"
+            control={control}
+            errors={errors}
+          />
+        </View>
+        <View style={styles.buttonWrapper}>
+          <Button label="Sign up" onPress={handleSubmit(onSubmit)} />
+        </View>
         <Link label="Go to Sign In" to={{ screen: RootScreenName.SIGN_IN }} />
```

Comment:
Remove this link, just move this screen under sing-in in navigation to be able to achieve it without changes in code